CMakeLists.txt

```diff
@@ -176,9 +176,18 @@ foreach(api ${nidrivers})
   endif()
 endforeach()
 
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+  set(SOURCE_OS
```

Comment:
How about `OSDEP_SOURCE`? Without looking at where this is used and what it's set to, I would expect a variable named `SOURCE_OS` to contain the name of an operating system.